modules/nodes-cluster-resource-configure-jdk.adoc

```diff
@@ -29,20 +29,20 @@ this documentation, and may involve setting multiple additional JVM options.
 == Understanding how to override the JVM maximum heap size
 
 For many Java workloads, the JVM heap is the largest single consumer of memory.
-Currently, the OpenJDK defaults to allowing up to 1/4 (1/`-XX:MaxRAMFraction`)
+Currently, the OpenJDK defaults to allowing up to 1/4 (1/`-XX:MaxRamPercentage`)
```

Comment:
This statement is incorrect, for "bare metal/ VM", the default is 1/4th, for "container (cgroups v1 and v2)", it should be like: ubi8: 50% (set via Xmx) ubi9: 80% (set via -XX:MaxRAMPercentage)* (refer to https://access.redhat.com/solutions/4886081 )

Reply:
Acknowledged.